src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -1253,4 +1255,28 @@ public function renderFieldAction($productId, $step, $fieldName)
             'formId' => $step . '_' . $fieldName . '_rendered',
         ]);
     }
+
+    /**
+     * @param Request $request
+     *
+     * @return Response
+     */
+    public function searchProductsAction(Request $request): Response
+    {
+        try {
+            $searchPhrase = $request->query->get('search_phrase');
+
+            /** @var FoundProduct[] $foundProducts */
+            $foundProducts = $this->getQueryBus()->handle(new SearchProducts($searchPhrase, 10));
```

Comment:
So for now you limit to 10 results and the pagination will be added in another issue?

Reply:
I just added 10 because it was no limit at all in legacy 😄 I'm not sure this needs a pagination as it could only show like 10 most relative results because its a search. Anyway It should be discussed with product team i guess.
(the pagination i was talking at the meeting was about carts/orders lists which aren't searchable)

Reply:
Ok, indeed it should be discussed with @MatShir